lighthouse-extension/gulpfile.js

```diff
@@ -104,9 +104,11 @@ gulp.task('chromeManifest', () => {
   .pipe(gulp.dest(distDir));
 });
 
-function applyBrowserifyTransforms(bundle) {
+function applyBrowserifyTransforms(bundle, channel) {
   // Fix an issue with imported speedline code that doesn't brfs well.
   return bundle.transform('./fs-transform', {global: true})
+  // Replace LH_CHANNEL enviroment variable with a string
+  .transform('./lh-channel-transform', {channel, global: true})
```

Comment:
This is a custom find&replace transform. I couldn't use envify because it's based on esprima 4 (latest) which doesn't support object rest/spread :(